Library/Homebrew/formulary.rb

```diff
@@ -343,6 +343,10 @@ def get_formula(spec, alias_path: nil, force_bottle: false, flags: [], ignore_er
     rescue FormulaClassUnavailableError => e
       raise TapFormulaClassUnavailableError.new(tap, name, e.path, e.class_name, e.class_list), "", e.backtrace
     rescue FormulaUnavailableError => e
+      if tap.core_tap? && Homebrew::EnvConfig.install_from_api?
+        raise CoreTapFormulaUnavailableError.new(name), "", e.backtrace
```

Comment:
What's the second argument here and why is it an empty string?

Reply:
Not sure, that's just what's done for the `raise TapFormulaUnavailableError` below. The goal is to mimick the `TapFormulaUnavailableError` as closely as possible with the `CoreTapFormulaUnavailableError` since they really mean the same thing (one is just a bit more specific)

Reply:
Gotcha, np then.